stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -351,6 +391,10 @@ ::util::Status BFChassisManager::PushChassisConfig(
 
 ::util::Status BFChassisManager::VerifyChassisConfig(
     const ChassisConfig& config) {
+  if (config.nodes_size() != 1) {
```

Comment:
I'm not sure if we support more than one node yet, or if there even is a chip of that kind.

We should probably adjust the error message.

Reply:
The code is set up to support multiple nodes, but there is an implicit mapping between Stratum node and SDK unit ID. That is, unit IDs are assigned sequentially based on the order of the nodes in the ChassisConfig which seems pretty brittle. Now does anyone use this? I don't know, but there definitely are vendors that build chassis switches or even 1U pizza boxes with more than one switching ASIC.

Reply:
And I dont know how it works or how the SDK handles it but there is a possibility to split the 2/4 pipelines from each other and configure them differently (as if they are different switches as I understood). It could be that the SDK gives the splitted pipelines different node id's....

Reply:
We should just check that the nodes_size() > 0

Reply:
Addressed in #405